plugins/modules/helm.py

```diff
@@ -676,6 +699,24 @@ def helmdiff_check(
     if reuse_values:
         cmd += " --reuse-values"
 
+    if reset_then_reuse_values:
+        helm_diff_version = get_plugin_version("diff")
+        helm_version = module.get_helm_version()
+        fail_msg = ""
+        if LooseVersion(helm_diff_version) < LooseVersion("3.9.12"):
+            fail_msg = "reset_then_reuse_values requires helm diff >= 3.9.12, current version is {0}\n".format(
+                helm_diff_version
+            )
+        if LooseVersion(helm_version) < LooseVersion("3.14.0"):
+            fail_msg += "reset_then_reuse_values requires helm >= 3.14.0, current version is {0}\n".format(
+                helm_version
+            )
+
+        if fail_msg:
+            module.fail_json(msg=fail_msg)
```

Comment:
A better way to concat those messages may be to use an array:

```suggestion
        fail_msg = []
        if LooseVersion(helm_diff_version) < LooseVersion("3.9.12"):
            fail_msg.append(
                "reset_then_reuse_values requires helm diff >= 3.9.12, current version is {0}\n".format(
                    helm_diff_version
                )
            )
        if LooseVersion(helm_version) < LooseVersion("3.14.0"):
            fail_msg.append(
                "reset_then_reuse_values requires helm >= 3.14.0, current version is {0}\n".format(
                    helm_version
                )
            )

        if fail_msg:
            module.fail_json(msg=", ".join(fail_msg))
```

Reply:
My bad, I didn't see the `\n` at the end of the comments.